postgis/mvt.c

```diff
@@ -310,7 +310,7 @@ static void parse_column_keys(mvt_agg_context *ctx)
 	char *key;
 	POSTGIS_DEBUG(2, "parse_column_keys called");
 
-	for (i = 0; i < natts; i++) {
+	for (i = 0; i < (uint32_t) natts; i++) {
```

Comment:
can natts be converted to uint32_t at assignment at start of function?

Reply:
Will do.